website/docs/components/tag/partials/code/component-api.md

```diff
@@ -5,22 +5,22 @@
     The text of the tag; or link text when the `@route` or `@href` are set. If no text value is defined an error will be thrown.
   </C.Property>
   <C.Property @name="href">
-    This is the URL parameter that is passed down to the `<a>` element.
+    URL parameter that’s passed down to the `<a>` element.
   </C.Property>
   <C.Property @name="isHrefExternal" @type="boolean" @values={{array "false" "true" }} @default="false">
-    This controls if the `<a>` link is external and so for security reasons we need to add the `target="_blank"` and `rel="noopener noreferrer"` attributes to it.
+    This controls if the `<a>` link is external. For security reasons, we add the `target="_blank"` and `rel="noopener noreferrer"` attributes to it by default.
   </C.Property>
   <C.Property @name="route models model query current-when replace">
-    These are the parameters that are passed down as arguments to the `<LinkTo>` / `<LinkToExternal>` components.
+    Parameters that are passed down as arguments to the `<LinkTo>`/`<LinkToExternal>` components.
   </C.Property>
   <C.Property @name="isRouteExternal" @type="boolean" @values={{array "false" "true" }} @default="false">
-    This controls if the "LinkTo" is external to the Ember engine ([more details here](https://ember-engines.com/docs/link-to-external)) in which case it will use a `<LinkToExternal>` instead of a simple `<LinkTo>` for the @route.
+    This controls if the “LinkTo” is external to the Ember engine, in which case it will use a `<LinkToExternal>` for the @route.
```

Comment:
```suggestion
    Controls if the “LinkTo” is external to the Ember engine, in which case it will use a `<LinkToExternal>` for the @route.
```
_Seems_ more consistent with the rest of the property descriptions that don't use "the"